rtf_vis_tool/src/stylesheets/Data_Association_PC.css

```diff
@@ -0,0 +1,45 @@
+/* Stylings for Data_Association_PC.js Component */
+
+.da-container {
```

Comment:
could we add comments explaining the choices for each of these css classes?

Reply:
done